src/program.js

```diff
@@ -298,6 +298,11 @@ Example: $0 --help run.
         type: 'string',
         coerce: coerceCLICustomPreference,
       },
+      'jsconsole': {
+        describe: 'Open web browser console.',
```

Comment:
this option would be a boolean because it doesn't take a value. It's passed without value like `web-ext run --jsconsole`.